.github/workflows/release.yaml

```diff
@@ -263,7 +269,7 @@ jobs:
         uses: "google-github-actions/setup-gcloud@v1"
 
       - name: Publish Helm Chart
-        if: ${{ !github.event.inputs.dry_run && !github.event.inputs.snapshot }}
+        if: ${{ !inputs.dry_run && !inputs.snapshot }}
```

Comment:
@deansheather we added this `if` in an earlier PR, but now I'm wondering. Were we supposed to upload helm charts for `snapshot = true`?

I'm asking because it'd publish a version like: `0.14.1-devel+dc91244b`.

Or I can drop the `!inputs.dry_run` which would allow us to publish on snapshot.

Reply:
I don't think so since this is just the publish step. We're still building and uploading the helm package to artifacts on dryrun/snapshot. We shouldn't be publishing any +dev versions to any official place (github releases, helm repo. docker repo etc.)

Reply:
Should we remove `snapshot`? I can't currently discern any difference in behavior between snapshot and dry-run. What is the difference supposed to be?

Reply:
since it implies dry run we have to refer to both everywhere, the only place where snapshot is used by itself is in the CODER_RELEASE env

Reply:
Mathias and I talked and we decided to remove snapshot and only have dry-run.